src/pages/DirectMessage/components/DMList/DMList.Styles.ts

```diff
@@ -56,7 +57,7 @@ export const DMListItemLayout = styled.button<{ $isSelect: boolean }>`
   justify-content: flex-start;
   align-items: flex-start;
   &:hover {
-    ${({ theme }) => theme.opacity.normal}
+    opacity: 0.5;
```

Comment:
잘못 수정된 것 같아서 다시 바꾸시면 될 것 같습니다 👍
```suggestion
    opacity: ${({ theme }) => theme.opacity.normal}
```